zoltar-tests/src/test/java/com/spotify/zoltar/FeatureExtractFnsTest.java

```diff
@@ -0,0 +1,47 @@
+/*-
+ * -\-\-
+ * zoltar-tests
+ * --
+ * Copyright (C) 2016 - 2018 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.zoltar;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+
+import com.google.common.collect.ImmutableList;
+import com.spotify.zoltar.FeatureExtractFns.BatchExtractFn;
+import com.spotify.zoltar.FeatureExtractFns.ExtractFn;
+import java.util.List;
+import org.junit.Test;
+
+public class FeatureExtractFnsTest {
+
+  @Test
+  public void batchExtractFn() throws Exception {
```

Comment:
Could you also provide example of usage?

Reply:
@ravwojdyla like one using the `FeatranExtractFns`?

Reply:
@regadas yeah - like having a full blown `Predictor`.

Reply:
@ravwojdyla added!